copy/compression.go

```diff
@@ -57,7 +57,7 @@ func blobPipelineDetectCompressionStep(stream *sourceStream, srcInfo types.BlobI
 		decompressor: decompressor,
 	}
 	if res.isCompressed {
-		res.srcCompressorName = format.Name()
+		res.srcCompressorName = format.InternalUnstableUndocumentedMIMEQuestionMark()
```

Comment:
- At which point do I put my feet down and say that this value must be documented properly before any new users are added? Surely no later than today.
- Reading the commit message, I don’t know what “since zstd and zstd:chunked would conflict in this case.” means.

(I’d also tentatively suggest that a set of very small PRs is more likely to be merged than one growing one.)

Reply:
This was removed from this PR. I vaguely remember we ended up with #2185 instead, for now.